src/main/java/net/atos/zac/app/admin/converter/RestBetrokkeneKoppelingenConverter.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.admin.converter
+
+import net.atos.zac.admin.model.BetrokkeneKoppelingen
+import net.atos.zac.admin.model.ZaakafhandelParameters
+import nl.info.zac.app.admin.model.RestBetrokkeneKoppelingen
+
+object RestBetrokkeneKoppelingenConverter {
```

Comment:
Why an object? No need for that at all I would say. I would remove this object completely and just make the functions pure functions, like we do elsewhere.